node/src/Pos/Client/Txp/Balances.hs

```diff
@@ -71,8 +73,12 @@ getOwnUtxosDefault addrs = do
 -- 1) It doesn't represent actual balances of addresses, but it represents _stakes_
 -- 2) Local utxo is now cached, and deriving balances from it is not
 --    so bad for performance now
-getBalanceDefault :: (MonadBalances m) => Address -> m Coin
-getBalanceDefault addr = getBalanceFromUtxo addr
+getBalanceDefault :: BalancesEnv ext ctx m => Address -> m Coin
+getBalanceDefault addr = do
+    balancesAndUtxo <- WS.getWalletBalancesAndUtxo
+    fromMaybe (mkCoin 0) .
+        HM.lookup addr .
```

Comment:
Introduced in https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58#diff-314075405d88168e3c4ea2e011625453